integration-tests/src/tests/client/features/congestion_control.rs

```diff
@@ -83,7 +84,7 @@ fn setup_contract(env: &mut TestEnv) {
 /// Simplest possible upgrade to new protocol with congestion control enabled,
 /// no traffic at all.
 #[test]
-fn test_protocol_upgrade() {
+fn test_protocol_upgrade_simple() {
```

Comment:
Ah because I'm lazy, it was easier to change the name than to figure out how to filter nextest to run only one test. Previously there were two tests where one was a prefix of another `test_protocol_upgrade` and `test_protocol_upgrade_under_congestion` and I couldn't run just the shorter one.

Reply:
I see, that makes sense.

Personally I just don't use nextest when running single tests... But if you want to do it, the syntax that I never remember is `-E 'test(=test_protocol_upgrade)`

https://nexte.st/book/running?highlight=exact#--skip-and---exact